src/reading.rs

```diff
@@ -260,6 +260,9 @@ impl Reader<BufReader<File>> {
     }
 }
 
+unsafe impl<T: Read + Seek> Send for Reader<T> {}
+unsafe impl<T: Read + Seek> Sync for Reader<T> {}
```

Comment:
Something like this would be more correct

```diff
diff --git a/src/encoding.rs b/src/encoding.rs
index 686e864..76b9e2c 100644
--- a/src/encoding.rs
+++ b/src/encoding.rs
@@ -51,7 +51,7 @@ impl_as_code_page_mark!(
 /// If the `yore` feature is on, this is implemented by all [`yore::CodePage`].
 ///
 /// Note: This trait might be extended with an `encode` function in the future.
-pub trait Encoding: EncodingClone + AsCodePageMark {
+pub trait Encoding: EncodingClone + AsCodePageMark + Send {
     /// Decode encoding into UTF-8 string. If codepoints can't be represented, an error is returned.
     fn decode<'a>(&self, bytes: &'a [u8]) -> Result<Cow<'a, str>, DecodeError>;

@@ -65,7 +65,7 @@ pub trait EncodingClone {

 impl<T> EncodingClone for T
 where
-    T: 'static + Encoding + Clone,
+    T: 'static + Encoding + Clone + Send,
 {
     fn clone_box(&self) -> Box<dyn Encoding> {
         Box::new(self.clone())
@@ -166,7 +166,7 @@ impl Encoding for DynEncoding {
 #[cfg(feature = "yore")]
 impl<T> Encoding for T
 where
-    T: 'static + yore::CodePage + Clone + AsCodePageMark,
+    T: 'static + yore::CodePage + Clone + AsCodePageMark + Send,
 {
     fn decode<'a>(&self, bytes: &'a [u8]) -> Result<Cow<'a, str>, DecodeError> {
         self.decode(bytes).map_err(Into::into)
@@ -194,7 +194,7 @@ where
 #[cfg(feature = "yore")]
 impl<CP> Encoding for LossyCodePage<CP>
 where
-    CP: 'static + yore::CodePage + Clone + AsCodePageMark,
+    CP: 'static + yore::CodePage + Clone + AsCodePageMark + Send,
 {
     fn decode<'a>(&self, bytes: &'a [u8]) -> Result<Cow<'a, str>, DecodeError> {
         Ok(self.0.decode_lossy(bytes))
--
```